x-pack/plugins/ml/public/jobs/jobs_list/components/multi_job_actions/group_selector/group_list/group_list.js

```diff
@@ -47,22 +48,65 @@ export class GroupList extends Component {
     this.state = {
       groups: [],
     };
+    // keep track of each of the group item refs
+    this.selectItems = [];
   }
 
   selectGroup = (group) => {
     this.props.selectGroup(group);
   }
 
+  moveUp = (event, index) => {
+    if (index < 0) {
+      return;
+    } else if (index > 0) {
+      event.preventDefault();
```

Comment:
should these `preventDefault` calls always fire anyway, rather than being contained within a condition?
Do we want the default action to happen if the user can't move up anymore? I'd suspect not.

Reply:
Hm, good point! I'll update it to preventDefault in both the moveUp and moveDown before we get into the conditional 👍